assignment/scripts/describe.js

```diff
@@ -159,7 +162,7 @@ if (temp > 39 || time >= 4) {
 /*
 let age = 21;
 const minAge = 21;
-
+// Change less than sign to greater than. 
```

Comment:
Not quite. With this change, someone who is 21 gets "no entry" when they should get "entry".